tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "helloapache",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "network": "BRIDGE",
+      "image": "$image",
+      "portMappings": [
+        {
+          "containerPort": 80,
+          "servicePort": $hostport,
```

Comment:
Yeah, you're right. LGTM. Maybe in the future we'll have valid JSON in case someone wants to convert / parse to yaml.